integration_test/gce/gce_testing.go

```diff
@@ -1351,7 +1355,7 @@ func CreateInstance(origCtx context.Context, logger *log.Logger, options VMOptio
 			// windows-*-core instances sometimes fail to be ssh-able: b/305721001
 			(IsWindowsCore(options.ImageSpec) && strings.Contains(err.Error(), windowsStartupFailedMessage)) ||
 			// SLES instances sometimes fail to be ssh-able: b/186426190
-			(IsSUSE(options.ImageSpec) && strings.Contains(err.Error(), startupFailedMessage)) ||
```

Comment:
This error message is very generic, so we may overhit this code path

Reply:
I created `IsSUSEImageSpec(imageSpec string) bool` that operates on the `ImageSpec` to keep the same logic.